ecosystem/indexer-grpc/indexer-grpc-utils/src/file_store_operator/local.rs

```diff
@@ -34,20 +47,16 @@ impl FileStoreOperator for LocalFileStoreOperator {
         }
     }
 
-    async fn get_transactions(&self, version: u64) -> anyhow::Result<Vec<String>> {
-        let batch_start_version = version / BLOB_STORAGE_SIZE as u64 * BLOB_STORAGE_SIZE as u64;
-        let current_file_name = generate_blob_name(batch_start_version);
-        let file_path = self.path.join(current_file_name);
+    fn storage_format(&self) -> StorageFormat {
+        self.storage_format
+    }
+
+    async fn get_transactions_bytes(&self, version: u64) -> anyhow::Result<Vec<u8>> {
```

Comment:
Can you name this get_transactions_old or uncompressed or something?

Reply:
done.